content/en/docs/ambient/install/multicluster/_index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Install Multicluster
+description: Install an Istio mesh in ambient mode across multiple Kubernetes clusters.
+weight: 40
+aliases:
+    - /docs/ambient/kubernetes/multicluster-install/
+    - /docs/ambient/kubernetes/multicluster/
+    - /docs/ambient/kubernetes/install/multicluster/
+    - /docs/ambient/install/multicluster/gateways/
+    - /docs/ambient/install/multicluster/shared/
+keywords: [kubernetes,multicluster]
+simple_list: true
+content_above: true
+test: table-of-contents
+owner: istio/wg-environments-maintainers
+---
+
+{{< boilerplate alpha >}}
+
+Follow this guide to install an Istio {{< gloss "ambient" >}}ambient service mesh{{< /gloss >}}
+that spans multiple {{< gloss "cluster" >}}clusters{{< /gloss >}}.
+
+## Current Status and Limitations
+
+{{< warning >}}
+**Ambient multicluster is currently in alpha status** and has significant limitations.
+This feature is under active development and should not be used in production environments.
+{{< /warning >}}
+
+Before proceeding with ambient multicluster installation, it's critical to understand
+the current state and limitations of this feature:
+
+### Supported Configurations
+
+Currently, ambient multicluster **only supports**:
+- **Multi-network topologies** with multiple primary clusters
+- **Double HBONE encapsulation** for cross-cluster traffic
```

Comment:
Sorry I'm a bit late to the party.

> - **Multi-network topologies** with multiple primary clusters
More accurately, each cluster has to be a primary cluster (have its own istiod). If you have 10 clusters but only two primary cluster, then you would have "multiply primary clusters" but this topology would not be supported.

> - **Double HBONE encapsulation** for cross-cluster traffic

Can we configure the this at all? is there any way for someone to try any other inner/outer tunnel protocol configuration. If not, then I think we can omit this.

Reply:
> More accurately, each cluster has to be a primary cluster (have its own istiod). If you have 10 clusters but only two primary cluster, then you would have "multiply primary clusters" but this topology would not be supported.

Totally agree - good clarification

> Can we configure the this at all? is there any way for someone to try any other inner/outer tunnel protocol configuration. If not, then I think we can omit this.

Even if we don't support other configurations, it might be good to explicitly clarify we currently don't support protocols other than double hbone. I could be convinced either way.

Reply:
I took a stab at rewording; let me know your thoughts